README.md

```diff
@@ -1,3 +1,22 @@
 # Helium Python Helper
 
 A helper module used across various Nebra repos to reduce redundant features.
+
+## utils
+
+### logger
+
+```python
+from hm_pyhelper.utils import logger
+logger = get_logger(__name__)
+logger.debug("message to log")
+```
+
+## Testing
+
+To run tests: 
+
+```bash
+pip install -r test-requirements
```

Comment:
Needs to be:

* `pip install -r requirements.txt`
* `pip install -r test-requirements.txt`

Reply:
Thanks, fixed: https://github.com/NebraLtd/hm-pyhelper/pull/14/commits/9d9d1d8728fb1f6c216d69a0aecb23ea14cdae27